test/RequestHandlerSwooleRunnerTest.php

```diff
@@ -33,7 +35,13 @@ public function setUp()
         $this->serverRequestError = function () {
             return $this->prophesize(ServerRequestErrorResponseGenerator::class)->reveal();
         };
-        $this->swooleHttpServer = $this->createMock(SwooleHttpServer::class);
+        $server = $this->createMock(Server::class);
```

Comment:
We're using [Prophecy](https://github.com/phpspec/prophecy) elsewhere for mocking, so please use that here.